docs/src/user-guide/core-clp-s.md

```
@@ -12,11 +12,27 @@ Usage:
+    timestamp.
+  * `--target-encoded-size` specifies the threshold in bytes for the size of the dictionaries and
+    encoded messages at which archives are split. This acts as a soft limit on memory usage for
+    compression, decompression, and search and also has a significant effect on compression ratio.
+  * `--structurize-arrays` specifies that arrays should be fully parsed and array entries should be
+    encoded into dedicated columns.
+  * `--auth <s3|none>` specifies the authentication method that should be used for network requests
+    if the input path is a URL. When S3 authentication is enabled we issue a GET request following
+    the presigned URL v4 specification. This request draws on the environment variables
+    `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and, optionally, `AWS_SESSION_TOKEN` if it exists.
+    For more information on usage with S3 see our
+    [dedicated guide](#guides-using-object-storage/index).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)